src/main/webapp/resources/js/pages/analysis/components/AnalysisError.jsx

```diff
@@ -9,10 +9,11 @@
 
 import React, { Suspense, useContext, useEffect, useState } from "react";
 import { AnalysisContext } from "../../../contexts/AnalysisContext";
-import { Alert, Spin, Tabs } from "antd";
```

Comment:
Create a `ContentLoading` component that can accept a message (maybe "Loading..." or something).  That way if we want to update it at any point we can do them all at once.

Reply:
Added in 5e42840cf8b9aa72a4ab4384c86a925e103d0382